backend/apps/owasp/signals/snapshot.py

```
@@ -0,0 +1,16 @@
+"""Snapshot signals."""
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from apps.owasp.models.snapshot import Snapshot
+from apps.owasp.utils.notifications import publish_snapshot_notification
+
+
+@receiver(post_save, sender=Snapshot)
+def snapshot_published(sender, instance, created, **kwargs):
```

Ruff (0.14.14)

[warning] Unused function argument: `sender`

(ARG001)